File: Nível 1/1546 - Feedback/main.lua
quantia = tonumber(io.read())

for i = 1, quantia do
    feedbacks = tonumber(io.read())

    for j = 1, feedbacks do 
        categoria = tonumber(io.read())

        if(categoria == 1) then
            print("Rolien")
        elseif(categoria == 2) then 
            print("Naej")
        elseif(categoria == 3) then 
            print("Elehcim")
        elseif(categoria == 4) then 
            print("Odranoel")
        end
    end
end

File: Nível 1/1546 - Feedback/main.py
quantia = int(input("A: "))

# 1 = Rolien
# 2 = Naej
# 3 = Elehcim
# 4 = Odranoel

for i in range(1, quantia + 1):
    feedbacks = int(input("B: "))
    for i in range(1, feedbacks + 1):
        categoria = int(input("C: "))

        if(categoria == 1):
            print("Rolien")
        elif(categoria == 2):
            print("Naej")
        elif(categoria == 3):
            print("Elehcim")
        elif(categoria == 4):
            print("Odranoel")